requirements.txt

```diff
@@ -1,2 +1,3 @@
 django-appconf
-django>=1.11.0
\ No newline at end of file
+django>=1.11.0
+django-health-check==3.0.0
```

Comment:
please don't add as a dependency. Make sure to mention it in the documentation. If should be a opt-in contrib feature.